src/vs/workbench/contrib/terminal/test/browser/xterm/xtermTerminal.test.ts

```diff
@@ -109,7 +109,7 @@ suite('XtermTerminal', () => {
 		instantiationService.stub(IViewDescriptorService, viewDescriptorService);
 
 		configHelper = instantiationService.createInstance(TerminalConfigHelper);
-		xterm = instantiationService.createInstance(TestXtermTerminal, Terminal, configHelper, 80, 30);
+		xterm = instantiationService.createInstance(TestXtermTerminal, Terminal, configHelper, 80, 30, 1);
```

Comment:
Should this be `, TerminalLocation.Panel` instead? Fine if so, let's just change in main